projects/laji/src/app/+observation/search-query.service.ts

```diff
@@ -87,7 +87,8 @@ export class SearchQueryService implements SearchQueryInterface {
     'atlasClass',
     'identificationBasis',
     'samplingMethod',
-    'taxonRankId'
+    'taxonRankId',
+    'hasValue'
```

Comment:
I added it to the search query service because the other query variables are there also, but I don't understand whether it's even used anywhere... It's a private readonly class property, and not used anywhere in the class so maybe it's dead code anyway? And I'm not sure if it should be "string" or "array" - it's comma separated string, like many other in the `array` list are.

Reply:
Ok thanks for the explanation, I didn't notice the usage in `forEachType` since it's referenced dynamically and hence not searchable by IDE. I removed it from the list

Reply:
Removing this broke the query, apparently it's needed even though it's not used in the observation search filters. I'll bring it back to the array type